### PR TITLE
Fix wrong aggregation of shap values

### PR DESCRIPTION
**Pull request by jvdd**
Fix a bug introduced in #49

### CODE REVIEW

**Comment by jvdd** [comment]
Fixes bug introduced in this line https://github.com/predict-idlab/powershap/pull/49#discussion_r1584839006

**Review by jonasvdd — APPROVED** [review]
LGTM

**Comment by jonasvdd** [comment]
@jvdd, maybe just add a comment / sanity check on which dimensions you might expect
You could also include an assert regarding the dimensions

File: powershap/shap_wrappers/shap_explainer.py [hunk 1]
```diff
@@ -171,11 +171,15 @@ def explain(
             Shap_values = np.abs(Shap_values)
 
             if len(np.shape(Shap_values)) > 2:
-                # Shap_values = np.max(Shap_values, axis=0)
-                Shap_values = np.max(Shap_values, axis=0).T
+                # SHAPE: (n_samples, n_features, n_outputs)
```
Comment (jonasvdd) [inline powershap/shap_wrappers/shap_explainer.py:174]: instead of `n_outputs` maybe `n_classes`?

Comment (JarneVerhaeghe) [reply powershap/shap_wrappers/shap_explainer.py:174]: voor multi-regression is het dan n_outputs

**Pull request merged**